From d97aaee81dc4b82595dc90ccb6a29bb0de3669d4 Mon Sep 17 00:00:00 2001
From: Matt Madison <matt@madison.systems>
Date: Fri, 5 Jun 2020 09:56:15 -0700
Subject: [PATCH] irq-gic: drop __init annotation from gic_init_fiq

It is called from a non-__init function, so it
should not have that annotation.

Signed-off-by: Matt Madison <matt@madison.systems>
---
 drivers/irqchip/irq-gic.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/irqchip/irq-gic.c b/drivers/irqchip/irq-gic.c
index a90a5841b20f..c3568461b9af 100644
--- a/drivers/irqchip/irq-gic.c
+++ b/drivers/irqchip/irq-gic.c
@@ -658,7 +658,7 @@ static struct irq_chip gic_chip = {
 };
 
 #ifdef CONFIG_FIQ
-static void __init gic_init_fiq(struct gic_chip_data *gic)
+static void gic_init_fiq(struct gic_chip_data *gic)
 {
 	void __iomem *dist_base = gic_data_dist_base(gic);
 
